### DISCUSSION TITLE
Kiosk mode

### DISCUSSION THREAD

**Discussion by sciurius** [home-assistant/frontend#16128] [Dashboard]
TLDR; Home Assistant should provide native kiosk functionality  in the core.

Due to the nature of Home Assistant functionality it is more that logical to show (frontend) dashboards on mobiles, tablets and other (often wall-mounted) panels. 

In many situations the sole purpose of the display is to show the information part of the dashboard only, most notably to **not** show things like sidebars and headings. This is often referred to as 'kiosk mode' for obvious reasons.

Surprisingly, Home Assistant does not provide a kiosk mode. Users who want this are directed to community and commercial 3rd party tools. See e.g. [Kiosk Mode in Home Assistant – How To – The Right Way!](https://www.youtube.com/watch?v=G3lT4zgjER8).

At the same time, Home Assistant clearly states that community tools are untested and not guaranteed to work in current or future releases. And indeed, the recent 2023 updates broke quite a few community cards and other tools.

While this may be 'part of the game' for many interesting functionality providing tools, it is hard to believe that this also affects functionality that really should be part of the Home Assistant core.
